content/variables/git.md

```diff
@@ -19,6 +19,8 @@ toc: false
 wip: false
 ---
 
+This variable shows metada about a specific piece of content. It is useful for times when you are using Git and Hugo to manage a project or in a multi-author setting, for example. It will not be useful as a cache busting method.
```

Comment:
\*metadata

That said, I think "metadata about a specific piece of content" is actually the description of front matter. I would instead recommend adding something to the effect of the following to the `note` admonition below instead:

> Hugo's Git integrations should be fairly performant but *can* increase build times for larger histories. As `.GitInfo` is specific to an individual content file, it is not an effective cache-busting method.